.ci/magic-modules/welcome-contributor.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo > comment/pr_comment << EOF
+Hello!  I am a robot who works on Magic Modules PRs.
+
+I have detected that you are a community contributor, so your PR will be assigned to someone with a commit-bit on this repo for initial review.  They will authorize it to run through our CI pipeline, which will generate downstream PRs.
+
+EOF
+
+shuf -n 1 <(printf "ndmckinley\nrambleraptor\nchrisst\nrileykarson\nSirGitsalot\nslevenick") > comment/assignee
```

Comment:
I'm not a fan of this (we'll forget to update it), but I'm assuming it's unavoidable. Is there a way to get this info from the GitHub API (getting a list of people from a team?)

Reply:
Gotta have a list someplace - the Github Graphite team has 42 members and maybe 1 in 5 of them is a regular contributor who checks their github notifications.